addons/l10n_fr_invoice_addr/views/report_invoice.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_invoice_document" inherit_id="sale.report_invoice_document_inherit_sale">
+        <xpath expr="//address" position="after">
+            <t t-set="partner" t-value="o.partner_id.parent_id or o.partner_id"/>
+            <t t-if="o.l10n_fr_is_company_french and partner.siret">
+                SIRET: <t t-esc="partner.siret"/>
+            </t>
+        </xpath>
+
+        <xpath expr="//address" position="attributes">
+            <attribute name="class">mb-0</attribute>
+        </xpath>
+
+        <xpath expr="//div[@id='informations']" position="inside">
+            <t t-if="o.l10n_fr_is_company_french and o.partner_id.parent_id">
+                <t t-set="partner" t-value="o.partner_id.parent_id"/>
```

Comment:
I think you want to use partner_id.commercial_partner_id everywhere, instead of partner.parent_id